charts/matrixbot/Chart.yaml

```
@@ -1,4 +1,4 @@
 description: Web3 Foundation bot for matrix.org
 name: matrixbot
-version: v1.0.1
+version: v1.0.2
```

I think this should be a minor version bump, new feature added.
```suggestion
version: v1.1.0
```